src/components/learner-credit-management/SpendTableEnrollmentDetails.jsx

```diff
@@ -12,17 +12,24 @@ const SpendTableEnrollmentDetailsContents = ({
   enterpriseSlug,
 }) => (
   <>
-    <EmailAddressTableCell row={row} />
+    <EmailAddressTableCell
+      tableId="spent"
+      userEmail={row.original.userEmail}
+      enterpriseEnrollmentId={row.original.enterpriseEnrollmentId}
+      fulfillmentIdentifier={row.original.fulfillmentIdentifier}
```

Comment:
[context] When the "Spent" table migrates its data source to pull from transactions list API via enterprise-subsidy, the row data will contain a `fulfillmentIdentifier`, not a `enterpriseEnrollmentId` though they have similar intents (i.e., uniquely identify some form of enrollment for the redemption). Proactively handling when the row either has the `enterpriseEnrollmentId` (analytics API) vs. the `fulfillmentIdentifier` (transactions API).